src/tfr_control/include/tfr_control/drivebase_publisher.h

```diff
@@ -0,0 +1,57 @@
+/****************************************************************************************
+ * File:            drivebase_publisher.h
+ * 
+ * Purpose:         This class is used by the drivebase node to convert from a Twist
+ *                  message in /cmd_vel to angular velocity for the two motor sets
+ *                  (left and right), handling the calculations for differential steering.
+ * 
+ *                  Every message read from /cmd_vel will result in two messages sent,
+ *                  one to each motor controller. This class will only publish after
+ *                  reading a message on /cmd_vel.
+ * 
+ * Subscribed To:   /cmd_vel
+ * Publishes To:    /left_tread_velocity_controller/command
+ *                  /right_tread_velocity_controller/command
+ ***************************************************************************************/
+#ifndef DRIVEBASE_PUBLISHER_H
+#define DRIVEBASE_PUBLISHER_H
+
+#include "ros/ros.h"
+#include "geometry_msgs/Twist.h"
+
+namespace tfr_control
+{
+    class DrivebasePublisher
+    {
+    public:
+        DrivebasePublisher() = delete;
+        explicit DrivebasePublisher(ros::NodeHandle& n, float axle_length, float wheel_radius);
```

Comment:
Do we want to include a definition of terms somewhere? I know for me at least "axle length" has been a confusing term (I mean, I know what it means now, but others might not: I initially thought it was the length of the axle holding the wheel on, which is like 12" long), and there's probably other confusing/ambiguous somewhere

Reply:
We shouldn't need a definition of terms if our naming choices are descriptive enough.  I'm changing the name to `wheel_span` to try to address this.  Is that more intuitive?

Reply:
Somewhat, I think `wheel_center_distance` might make a bit more sense. It's kinda a weird term to describe in a variable name